tests/engine/models/test_actor_model.py

```diff
@@ -17,12 +18,45 @@ def test_actor_proper_shape():
     K = 3
     I = 100
 
+    return N, SE, BATCH_SIZE, max_N, UE, K, I
+
+
+def test_actor_proper_shape(params):
+    N, SE, BATCH_SIZE, max_N, UE, K, I = params
+
     example_user = torch.rand(BATCH_SIZE, UE)
     example_services_history = torch.rand(BATCH_SIZE, N, SE)
     example_mask = torch.ones(BATCH_SIZE, I)
-
     example_state = (example_user, example_services_history, example_mask)
 
     actor = Actor(K, SE, UE, I, MLPHistoryEmbedder(SE, max_N))
+    actor_output = actor(example_state)
+
+    assert (actor.act_min < actor_output).all()
+    assert (actor.act_max > actor_output).all()
+
+    assert actor_output.shape == torch.Size([BATCH_SIZE, K, SE])
+
+
+def test_actor_min_max(params):
+    N, SE, BATCH_SIZE, max_N, UE, K, I = params
+
+    example_user = torch.rand(BATCH_SIZE, UE)
+    example_services_history = torch.rand(BATCH_SIZE, N, SE)
+    example_mask = torch.ones(BATCH_SIZE, I)
+    example_state = (example_user, example_services_history, example_mask)
 
-    assert actor(example_state).shape == torch.Size([BATCH_SIZE, K, SE])
+    act_boundaries = [(-1, 1), (8.0, 10.0), (-5, -4), (0.75, 1.25)]
```

Comment:
Maybe we can randomize those ranges a little bit? Perhaps using https://hypothesis.readthedocs.io/en/latest/?